server/build/plugins/nextjs-ssr-import.js

```diff
@@ -10,13 +10,15 @@ export default class NextJsSsrImportPlugin {
 
   apply (compiler) {
     compiler.plugin('compilation', (compilation) => {
-      compilation.mainTemplate.plugin('require-ensure', (code) => {
+      compilation.mainTemplate.plugin('require-ensure', (code, chunk) => {
         // Update to load chunks from our custom chunks directory
-        const chunksDirPath = join(this.dir, this.dist, 'dist')
+        const baseDirPath = join(this.dir, this.dist, 'dist')
+        const pagePath = join(this.dir, this.dist, 'dist', dirname(chunk.name))
```

Comment:
I wonder if we can use `output.path` from the webpack config here, it's the path to `join(this.dir, this.dist, 'dist')`

Reply:
BTW: What's `output`?

Reply:
`compiler.options.output.path`

Reply:
Ah ha. You mean passing the outputPath to the constructor.
That's good.

Reply:
https://github.com/zeit/next.js/blob/canary/server/build/webpack.js#L134

Reply:
We actually don't have to even pass it to the constructor. It's already available. And since chunks are always compiled to the output.path + name, it's reasonable to read it from there 🙏

Reply:
And it's more future proof, since if we ever change the outputpath we'd also have to change the path in this plugin, which is unneeded 👍